build.sbt

```diff
@@ -67,9 +68,13 @@ val core = crossProject(JVMPlatform, JSPlatform)
   .in(file("modules/core"))
   .settings(moduleSettings)
   .settings(
+    name := "cats-retry",
```

Comment:
This will set the module name to `cats-retry-cats-retry` 😄 

Let's remove the `moduleName := ...` line from `moduleSettings` above.